### DISCUSSION TITLE
Error on installing the requirements

### DISCUSSION THREAD

**Discussion by cebause01** [MHProDev/MHDDoS#526] [Q&A]
error: could not open 'impacket\examples\serviceinstall.py': Invalid argument
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for impacket
  Running setup.py clean for impacket
  Building wheel for pyroxy (setup.py) ... done
  Created wheel for pyroxy: filename=PyRoxy-1.0b5-py3-none-any.whl size=2830064 sha256=752490e026b2bae67054cd9e6ab5a9ee3801d35f4756bba284dd4b22ece73fcc
  Stored in directory: C:\Users\User\AppData\Local\Temp\pip-ephem-wheel-cache-7y_c4ypd\wheels\e1\61\9d\1d12d80e29e1551fa4e9b7f0778b42fcce18e9ab52d3a2be62
Successfully built pyroxy
Failed to build impacket
ERROR: Could not build wheels for impacket, which is required to install pyproject.toml-based projects

**Comment by Lefoxxxx** [comment]
Disable Windows Defender
